src/app/carbon-estimation-table/carbon-estimation-table.component.html

```diff
@@ -0,0 +1,81 @@
+<!-- eslint-disable @angular-eslint/template/interactive-supports-focus -->
+<table role="treegrid" aria-label="Estimation of emissions" class="tce-table-fixed tce-w-full">
+  <thead>
+    <tr class="tce-border-b-2">
+      <th class="tce-text-left">Category</th>
+      <th class="tce-w-40 tce-text-left">Emissions</th>
+    </tr>
+  </thead>
+  @if (carbonEstimation()) {
+    <tbody
+      (keydown.home)="homeEndKeyBoardEvent($event, true)"
```

Comment:
Could these functions pull the key information out of `KeyboardEvent.key`? These seem like they'd be easy to set up wrong, especially the booleans where you don't really have any context for what it means.

Reply:
Yeah they could, will update